README.md

```diff
@@ -105,7 +153,11 @@ d4p.daalfini()
 
 daal4py patching affects performance of specific Scikit-learn functionality listed below. In cases when unsupported parameters are used, daal4py fallbacks into stock Scikit-learn. These limitations described below. If the patching does not cover your scenarios, [submit an issue on GitHub](https://github.com/IntelPython/daal4py/issues).
 
-Scenarios that are already available in 2020.3 release:
+⚠️ We support optimizations for the last 4 versions of Scikit-learn. The latest release of daal4py-2021.1 suppors Scikit-learn 0.21.X,
```

Comment:
Do we really want to list all versions instead of putting statement that last 4 supported? 
It will be required to update this list with every scikit release.

Reply:
I'm not sure. It seems to me that we need to clearly state what we support, so that there is no question. Can we automate this?

Reply:
Let's keep for now

Reply:
```suggestion
⚠️ We support optimizations for the last four versions of scikit-learn. The latest release of daal4py-2021.1 supports scikit-learn 0.21.X,
```